### PR TITLE
Added new terms, FAQ schema, and meta description

### PR DESCRIPTION
**Pull request by CathPag**

### CODE REVIEW

File: linkerd.io/content/service-mesh-glossary/_index.md [hunk 1]
```diff
@@ -2,73 +2,83 @@
...
+      Load balancing is the act of distributing work across a number of 
+      equivalent endpoints. Kubernetes, like many systems, provides load 
+      balancing at the connection level. A service mesh like Linkerd improves 
+      this by performing load balancing at the request level.
+  - question: What is multi-cluster?
+    answer:
+      In the context of Kubernetes, multi-cluster usually refers to running 
+      an application "across" multiple Kubernetes clusters. Linkerd's 
+      multi-cluster support provides seamless and secured communication 
+      across clusters, in a way that's secure even across the open Internet.
```
Comment (cpretzer) [inline linkerd.io/content/service-mesh-glossary/_index.md:62]: what do you think of replacing `open` with `public`?

File: linkerd.io/content/service-mesh-glossary/_index.md [hunk 2]
```diff
@@ -115,6 +134,36 @@ Linkerd was the first service mesh and the project that defined the term
...
+called EWMA (exponentially-weighted moving average), to distribute load
+based on endpoint latency while being responsive to rapid changes in the
+latency profile of individual endpoints.
+
+## Multi-cluster
+
+In the context of Kubernetes, multi-cluster usually refers to running
+an application "across" multiple Kubernetes clusters. Linkerd's multi-cluster
+support provides seamless and secured communication across clusters, in a
+way that's secure even across the open Internet, and is fully transparent
```
Comment (cpretzer) [inline linkerd.io/content/service-mesh-glossary/_index.md:165]: @CathPag See note above about replacing `open` with `public`

**Review by cpretzer — COMMENTED** [review]
Just a couple of small notes

File: linkerd.io/content/service-mesh-glossary/_index.md (hunk 1 above)
Comment (CathPag) [reply linkerd.io/content/service-mesh-glossary/_index.md:62]: sure

File: linkerd.io/content/service-mesh-glossary/_index.md (hunk 2 above)
Comment (CathPag) [reply linkerd.io/content/service-mesh-glossary/_index.md:165]: ok

File: linkerd.io/content/service-mesh-glossary/_index.md (hunk 1 above)
Comment (cpretzer) [reply linkerd.io/content/service-mesh-glossary/_index.md:62]: Great! Please make the changes and then we can get this merged :)

**Review by cpretzer — CHANGES_REQUESTED** [review]

**Comment by CathPag** [comment]
done

**Review by cpretzer — APPROVED** [review]

**Pull request merged**